tests/DsnTest.php

```diff
@@ -29,6 +29,28 @@ public function testMySql()
         $this->assertEquals('root_pass', $auth['password']);
     }
 
+    public function testSqlite()
+    {
+        $inputString = 'sqlite:///root:root_pass@127.0.0.1/test.db';
```

Comment:
This is not a valid DSN. 

`sqlite:///var/test.db` is valid and `sqlite://root:root_pass@127.0.0.1/test.db` is also valid. 

Im almost sure Im correct. I also tried searching in sqlite docs but I could not find anything there related to this issue.